source/buildings/sawmill.md

```diff
@@ -16,12 +16,12 @@ layout: default
         </div>
     </div>
     <hr />
-    <recipe>sawmill</recipe>
+    <recipe>Lumberjack's Hut</recipe>
```

Comment:
This part would need to be reverted. Updating this without changing the recipe file's name would break the recipe on this page. I'll have to change it when I can get round to updating the recipes.